src/AWS.Logger.Core/AWSLoggerConfig.cs

```diff
@@ -129,13 +129,23 @@ public AWSLoggerConfig(string logGroup)
         /// <summary>
         /// Gets and sets the LogStreamNameSuffix property. The LogStreamName consists of a DateTimeStamp as the prefix and a user defined suffix value that can 
```

Comment:
This documentation should be changed to match the other summaries (i.e. add info about the optional custom prefix).

Reply:
Fixed